src/TransactionsSummary/AmountSummary/AmountSummary.jsx

```diff
@@ -0,0 +1,14 @@
+import classes from './styles.module.css';
+export function AmountSummary({ transactions }) {
+	const total = transactions.reduce(
+		(total, currentAmount) => (total = total + currentAmount.amount),
+		0
+	);
+	return (
+		<tr>
```

Comment:
takie podejscie jest niebezpieczne. Tworzysz tutaj ukryta zaleznosc.
Zakladasz ze ten komponent bedzie wyswietlany w ramach tabeli.
(Uzycie <tr> poza tabela jest niewskazane).

Zamiast tego lepiej opakowac to w div/albo paragraf i w miejscu gdzie tego uzywasz przygotowac odpowiednia strukture. Mozemy sie zdzwonic to Ci pokaze na zywo o co chodzi